README.md

```diff
@@ -57,10 +57,15 @@ dependencies {
 
 *API key* must be provided, otherwise every `StSDK` get method call will result in an error.
 
-**Java/Kotlin:**
+**Kotlin:**
```

Comment:
I do not understand these changes.
Please, only stirp the /Kotlin from the readme. Otherwise I want to stick with Java examples, since it's more known. Second, the example showed hot to create an instance. Not how to obtain it. Third, the getting sdk is implementation detail of demo app and should not affect the readme.

Reply:
:heavy_check_mark: